apps/fares/test/format_test.exs

```diff
@@ -27,7 +27,7 @@ defmodule Fares.FormatTest do
       assert media(%Fare{media: [:charlie_card]}) == "CharlieCard"
       assert media(%Fare{media: [:commuter_ticket]}) == "CharlieTicket"
       assert media(%Fare{media: [:mticket]}) == "mTicket App"
-      assert media(%Fare{media: [:cash]}) == "Cash"
```

Comment:
🙃 uh oh... let's leave this as failing (in this PR) and update the source code to correctly output this as capitalized (in a later ticket).

Reply:
It was intended.  It wasn't caught in the ticket that required the word cash to be lower case.
I can move it to its own pull request